tests/integration/support/client_wrapper.go

```diff
@@ -0,0 +1,202 @@
+package support
+
+import (
+	"fmt"
+	"io/ioutil"
+	"log"
+	"os"
+	"path"
+	"path/filepath"
+	"strings"
+
+	"github.com/optimizely/go-sdk/pkg"
+	"github.com/optimizely/go-sdk/pkg/client"
+	"github.com/optimizely/go-sdk/pkg/config"
+	"github.com/optimizely/go-sdk/pkg/decision"
+	"github.com/optimizely/go-sdk/pkg/entities"
+	"github.com/optimizely/go-sdk/pkg/event"
+	"github.com/optimizely/go-sdk/tests/integration/models"
+	"github.com/optimizely/go-sdk/tests/integration/optlyplugins"
+	"gopkg.in/yaml.v3"
+)
+
+// ClientWrapper - wrapper around the optimizely client that keeps track of various custom components used with the client
+type ClientWrapper struct {
+	Client          *client.OptimizelyClient
+	DecisionService decision.Service
+	Config          pkg.ProjectConfig
```

Comment:
Do you need to keep track of this separately? You can instead call `Client.GetProjectConfig()` whenever you need to so you don't have to retain a reference here